redhat-cop/casl-ansible
# Use the correct variable name in create_users

The user creation was failing, because it was looking for the
`demo_users` variable while the samples put the data under
`create_users`.

#### What does this PR do?

It makes the `create_users` section in the sample inventory actually create the users.

#### How should this be manually tested?

Add the following to `OSv3.yaml`, run the end-to-end playbook and verify you can log in with username `rdu-user00` and password `rdu-sample`:

```yaml
create_users:
  num_users: 5
  prefix: 'rdu-user'
  passwd_file: '/etc/origin/master/htpasswd'
  password: 'rdu-sample'
```

#### Is there a relevant Issue open for this?

No

#### Who would you like to review this?
cc: @redhat-cop/casl
